content/blog/20201119-models-of-society.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "Models of Society"
+description = "Conversations compose society. What composes conversation&mdash;how do we digitize it in a way that enhances society without imposing upon it? How do we form this new medium, both to facilitate natural human behavior and to inspire the best of it?"
+date = 2020-11-19
+[extra]
+author = "Tyler Shuster"
+ship = "~radbur-sivmus"
+image = "https://media.urbit.org/site/posts/essays/cochno-stone.png"
++++
+
+<br>
+
+![Ludovic Maclellan Mann examining the Cochno Stone](https://media.urbit.org/site/posts/essays/cochno-stone.png)
```

Comment:
Can we move this back under "Modes of Messaging" ? I know it's not exactly convention